pcsx2/ImGui/ImGuiFullscreen.cpp

```diff
@@ -992,9 +992,41 @@ void ImGuiFullscreen::SetFullscreenFooterText(std::span<const std::pair<const ch
 	CreateFooterTextString(s_fullscreen_footer_text, items);
 }
 
+void ImGuiFullscreen::AppendToFullscreenFooterText(std::span<const std::pair<const char*, std::string_view>> items)
+{
+	if (s_fullscreen_footer_text.empty())
+	{
+		CreateFooterTextString(s_fullscreen_footer_text, items);
+		return;
+	}
+
+	SmallStringBase& dest = s_fullscreen_footer_text;
+	for (const auto& [icon, text] : items)
+	{
+		if (!dest.empty())
```

Comment:
Given that an empty `dest` is assigned `s_fullscreen_footer_text` and an empty `s_fullscreen_footer_text` is already handled above, surely this `dest` is never empty here (and thus this `if` statement is always true)?